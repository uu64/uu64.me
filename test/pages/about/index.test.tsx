import React from "react";
import renderer from "react-test-renderer";
import App from "../../../pages/about/index.mdx";

describe("App", () => {
  it("renders correctly", async () => {
    const component = renderer.create(<App />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
