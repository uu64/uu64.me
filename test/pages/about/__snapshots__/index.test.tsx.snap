// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  className="about-content px-8"
>
  <div
    className="pt-8 pb-2"
  >
    <h1
      className="font-semibold text-3xl text-custom-normal"
    >
      About
    </h1>
    <p
      className="border-b-2 border-gray-400 py-2 text-sm"
    >
      このサイトについて
    </p>
  </div>
  <h3
    className="pt-8 pb-4 font-semibold"
  >
    筆者について
  </h3>
  <div
    className="flex justify-center"
  >
    <img
      alt="profile"
      className="rounded-full p-4"
      src="icon.jpg"
      style={
        Object {
          "height": 130,
          "width": 130,
        }
      }
    />
  </div>
  <h2
    className="text-center font-bold pb-4"
  >
    uu
  </h2>
  <p>
    某メーカーのソフトウェアエンジニア。
    <br />
    最近の仕事では AWS CloudFormation を活用した IaC や CI/CD パイプラインの実装を担当することが多い。
    <br />
    <br />
    主な技術スタック： JavaScript, Java, Amazon Web Services, Git, Linux
  </p>
  <h3
    className="pt-8 pb-4 font-semibold"
  >
    アクセス解析について
  </h3>
  <p
    className="leading-relaxed"
  >
    このサイトでは、コンテンツの分析と改善を目的としてGoogle Analyticsを利用しています。
  </p>
  <p
    className="leading-relaxed"
  >
    Google Analyticsはトラフィックデータの収集のためにCookieを使用しています。
このトラフィックデータは匿名で収集されており、個人を特定するものではありません。
詳細については

    <a
      className="text-custom-normal"
      href="https://www.google.com/intl/ja/policies/privacy/partners/"
      rel="noopener noreferrer"
      target="_blank"
    >
      「ユーザーが Google パートナーのサイトやアプリを使用する際の Google によるデータ使用」のページ
    </a>
    
を参照してください。
  </p>
</div>
`;
